src/unix/fs.c

```diff
@@ -229,9 +229,46 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 #endif
 }
 
+static ssize_t uv__fs_access(uv_fs_t * req) {
+#ifdef __MVS__
+  int r = 0;
+
+  if (req->flags | F_OK)
```

Comment:
I think we should leave that out of libuv. fs APIs are pretty low level, I don't think we do this kind of mangling elsewhere.